atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownTestSuite.java

```diff
@@ -0,0 +1,133 @@
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+import com.google.common.collect.ImmutableMap;
+import com.jayway.awaitility.Awaitility;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.cassandra.CassandraKeyValueServiceConfigManager;
+import com.palantir.atlasdb.containers.Containers;
+import com.palantir.atlasdb.containers.ThreeNodeCassandraCluster;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool;
+import com.palantir.atlasdb.keyvalue.cassandra.CassandraKeyValueService;
+import com.palantir.docker.compose.connection.Container;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        OneNodeDownGetTest.class,
+        OneNodeDownPutTest.class,
+        OneNodeDownMetadataTest.class,
+        OneNodeDownDeleteTest.class,
+        OneNodeDownTableManipulationTest.class
+        })
+public final class OneNodeDownTestSuite {
+
+    private static final String CASSANDRA_NODE_TO_KILL = ThreeNodeCassandraCluster.FIRST_CASSANDRA_CONTAINER_NAME;
+
+    public static final TableReference TEST_TABLE = TableReference.createWithEmptyNamespace("test_table");
+
+    public static final byte[] FIRST_ROW = "row1".getBytes();
+    public static final byte[] SECOND_ROW = "row2".getBytes();
+    public static final byte[] FIRST_COLUMN = "col1".getBytes();
+    public static final byte[] SECOND_COLUMN = "col2".getBytes();
+    public static final Cell CELL_1_1 = Cell.create(FIRST_ROW, FIRST_COLUMN);
+    public static final Cell CELL_1_2 = Cell.create(FIRST_ROW, SECOND_COLUMN);
+    public static final Cell CELL_2_1 = Cell.create(SECOND_ROW, FIRST_COLUMN);
+    public static final Cell CELL_2_2 = Cell.create(SECOND_ROW, SECOND_COLUMN);
+    public static final Cell CELL_3_1 = Cell.create("row3".getBytes(), FIRST_COLUMN);
+    public static final Cell CELL_3_2 = Cell.create("row3".getBytes(), SECOND_COLUMN);
+
+    public static final byte[] DEFAULT_VALUE = "default_value".getBytes();
+    public static final long DEFAULT_TIMESTAMP = 2L;
+
+    public static CassandraKeyValueService db;
+
+    private OneNodeDownTestSuite(){}
+
+    @ClassRule
+    public static final Containers CONTAINERS = new Containers(OneNodeDownTestSuite.class)
+            .with(new ThreeNodeCassandraCluster());
+
+    @BeforeClass
+    public static void initializeKvsAndDegradeCluster() throws IOException, InterruptedException {
+        setupTestTable();
+        degradeCassandraCluster();
+        db = createCassandraKvs();
+    }
+
+    @AfterClass
+    public static void bringClusterBack() throws IOException, InterruptedException {
+        db.close();
+    }
+
+    private static void setupTestTable() {
+        db = createCassandraKvs();
```

Comment:
Suggest using a local variable rather than manipulating `db`. The fewer side effects of a method, the better: you've made the class-order dependent here.